### PR TITLE
[LED-97] Duplicate Transaction Popup Allows editing.

### PR DESCRIPTION
**Pull request by cjm721**

### CODE REVIEW

**Comment by cjm721** [comment]
Currently Looking for testing off the Add Transaction refactors.

# This PR is not ready for approval.

**Comment by cjm721** [comment]
PR is now ready for review.

File: src/main/java/ledger/user_interface/ui_controllers/UserTransactionInput.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.ResourceBundle;
+
+/**
+ * Created by CJ on 11/9/2016.
+ */
+public class UserTransactionInput extends GridPane implements IUIController, Initializable {
```
Comment (shellajt) [inline src/main/java/ledger/user_interface/ui_controllers/UserTransactionInput.java]: Lets document this a bit

File: src/main/resources/fxml_files/DuplicateTransactionPopup.fxml [hunk 2]
```diff
@@ -1,27 +1,54 @@
...
+      <GridPane GridPane.rowIndex="2">
+        <columnConstraints>
+          <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+          <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
+        </columnConstraints>
+        <rowConstraints>
+          <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+        </rowConstraints>
+         <children>
+            <Button fx:id="importButton" mnemonicParsing="false" text="Import" textAlignment="CENTER" GridPane.columnIndex="1" GridPane.halignment="RIGHT">
```
Comment (shellajt) [inline src/main/resources/fxml_files/DuplicateTransactionPopup.fxml]: Lets change these buttons to be consistent with our others. I think the color is just changed to create that orange box look

**Comment by LadyMopar** [comment]
Ditto on being consistent with the buttons, and also the white background needs to extend behind them. There should also be some sort of margin around them so they're not sitting on the bottom of the window.

File: src/main/java/ledger/user_interface/ui_controllers/UserTransactionInput.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            return null;
+        }
+
+        Transaction t = new Transaction(date, type, amount, account,
+                payee, pending, category, notes);
+
+        return t;
+    }
+
+    public void setTransaction(Transaction currentTrans) {
```
Comment (shellajt) [inline src/main/java/ledger/user_interface/ui_controllers/UserTransactionInput.java]: Javadoc

**Pull request merged**